fixtures/chauvet-professional/kolorado-quad-zoom.json

```
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Kolorado Quad zoom",
+  "shortName": "Kolorado",
```

```suggestion
  "name": "COLORado 2-Quad Zoom",
  "shortName": "COLORado",
```

Coincidentally, OFL already has a fixture called "Kolorado"!